packages/core/src/initializer/GitignoreWriter.ts

```diff
@@ -0,0 +1,23 @@
+import * as os from 'os';
+
+import { tokens } from '@stryker-mutator/api/plugin';
+import { fsAsPromised } from '@stryker-mutator/util';
+
+import { initializerTokens } from '.';
+
+const GITIGNORE_FILE = '.gitignore';
+
+export default class GitignoreWriter {
+  public static inject = tokens(initializerTokens.out);
+  constructor(private readonly out: typeof console.log) {}
+
+  public async addStrykerTempFolder() {
+    if (fsAsPromised.existsSync(GITIGNORE_FILE)) {
+      const strykerTempFolderSpecification = `${os.EOL}# stryker temp files${os.EOL}*.stryker-tmp${os.EOL}`;
```

Comment:
Please define ` '.stryker-tmp'` as a const to be reused (from `@stryker-mutator/api/config`), making it DRY

Reply:
Also: running stryker init multiple times will result in multiple rules being added to your .gitignore file. We can do a simple `if(!gitignoreContent.endswith(strykerTempFolderSpecification)` to prevent this. Not really necessary maybe, but nice none the less.

Reply:
Why `*.stryker-tmp` instead of `.stryker-tmp`?